### DISCUSSION TITLE
Custom Texture/Models

### DISCUSSION THREAD

**Discussion by Marrrcs** [DashieDev/DoggyTalentsNext#15] [Q&A] [answered]
So to add a custom texture I'm assuming i can just add it to the mods .jar assets? Is there an optimal way to do this? And what would happen if i would like to use a smaller model.

**Comment by DashieDev** [comment, marked as answer]
Before 1.11.0 it is true.
You can just place your texture directly in the jar.
at here ```assets\doggytalents\textures\entity\dog\custom```
But a recommended way would be to add your own resource pack so you can share it if you want.

1.11.0 the skin system has undergone an upgrade that supports different model make way for Optifine CEM support in the future.
You now also have to edit the ```skin.json``` file to include your skin in 😅

**Reply by Marrrcs** [reply]
Just to make sure I'm understanding correctly, It's possible to add the model in as of right now or that will be in the future? Btw I am trying to make this work to add a new dog in, is it possible to know what the model name is and if it's possible you could send me the model or tell me where i can find it?

**Reply by DashieDev** [reply]
It will be a future project as it requires me to implement a JEM parser myself 😅 which I make some progress on already. There are about like three additional models that i programmatically added to demonstrate the possibility of having different model rendered.
